projects/bp-gallery/src/components/views/archives/ArchiveEditView.tsx

```diff
@@ -20,13 +20,17 @@ interface ArchiveEditViewProps {
   archiveId: string;
 }
 
+interface PaypalProperties {
+  paypalClient?: string;
+  paypalDonationText?: string;
+  paypalPurpose?: string;
```

Comment:
- I think this reads better if you remove the `paypal` prefixes
- The optional types shouldn't be necessary here, see my comment on the `updateForm` calls